pkg/tsdb/elasticsearch/response_parser.go

```diff
@@ -939,3 +1006,13 @@ func createFieldOfType[T int | float64 | bool | string](docs []map[string]interf
 	field.Config = &data.FieldConfig{Filterable: &isFilterable}
 	return field
 }
+
+func setPreferredVisType(frame *data.Frame, visType data.VisType) {
+	if frame.Meta != nil {
```

Comment:
nitpick: i'd write this as:
```go
if frame.Meta == nil {
    frame.Meta = &data.FrameMeta{}
}

frame.Meta.PreferredVisualization = visType
```

but that's just personal preference, no stong feelings there.

```

Reply:
MUCH more readable! Changed in https://github.com/grafana/grafana/pull/63647/commits/e92a065fe27bfc43de47ed855292e71369ac497f.